### ISSUE TITLE
Transform labels classes into labels

### ISSUE THREAD

**Issue by Sphinxs** [ThilinaRajapakse/simpletransformers#901]
I have a problem that needs multi-label and multi-class classification to be solved, for this purpose, I have codified my labels classes into ranges 0...N, the problem its that the current version *0.51.3* of Simple Transformers [don't support](https://simpletransformers.ai/docs/classification-data-formats/#multi-label-classification) multi-label and multi-class classification at once, i.e you can either set 0 or 1 for each label, which makes you take a decision on how to solve your problem: multiple labels or multiple classes.

Get all unique classes on each label and turn them into labels, where a filter is applied to check if that class (that now became a label) appears in row 0..N, something like:

```python
model_labels = DataFrame()

for column in data_labels.columns:
    for value in data_labels[column].unique():      
        print(column, value, end='\n')
        
        model_labels = concat([model_labels, DataFrame(data=(data_labels[column] == value).astype(int).to_list(), columns=[value])], axis=1)
    
model_labels.head()
``` 

I considered One Vs Rest, here and when I was considering Tree -> Decision Tree, Random Forest -> AdaBoost -> XGBoost -> CaTboost -> LightGBM, etc; Label Encoder + Multi Label Binarizer + One Hot Encoder (also though about Glove, Word 2 Vec... but no!) and other small things. Before I get into Attention, Transformers and so on I tested Auto ML and the same problem was there, but a little bit worse cause I didn't have control.

What I'm proposing will not fix the problem or be the best solution, but it can be an alternative for quick tests (even more if the support for more than one sentence is added as a feature to the model). I also implemented a fine-tune version using Bert and Roberta through Hugging Face and Transformers and that's why I don't think this is the best solution, by seeing how the tokenizer works...
